scripts/enable_lambdas.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import ast
+from azul import config
+from azul.types import JSON
+import boto3
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class RedButton:
+    def __init__(self):
+        self.azul_lambda = boto3.client('lambda')
```

Comment:
`azul_lambda` sounds weird. The convention is to append an underscore if a conflict with a built-in arises.

```suggestion
        self.lambda_ = boto3.client('lambda')
```